### DISCUSSION TITLE
deta login fail

### DISCUSSION THREAD

**Discussion by jpzoll** [deta/cloud-docs#233] [Help]
Hello! I got the following message when trying to log into deta from the terminal for the first time.

<img width="1138" alt="image" src="https://user-images.githubusercontent.com/55609216/171283661-649c7b46-8c4d-454f-9945-caf2069cc5f7.png">

I don't intend on using anything other than Safari. Can I still use deta?

**Comment by rohanshiva** [comment]
You can use Access Tokens, please refer to [these docs](https://docs.deta.sh/docs/cli/auth/#deta-access-tokens) to see how they work.
